LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2009-2016 Homebrew contributors.
+Copyright 2009-2017 Homebrew contributors.
```

Comment:
I'm personally fine with leaving it as Homebrew. I'd be willing to accept a PR to add
`Copyright 2009-present Homebrew and Linuxbrew contributors.`